### PR TITLE
Support multiple source paths.

### PR DESCRIPTION
**Pull request by zeroify**

### CODE REVIEW

File: packages/create-react-kotlin-app/createReactApp.js [hunk 1]
```diff
@@ -63,6 +78,12 @@ const program = new commander.Command(packageJson.name)
     '--scripts-version <alternative-package>',
     'use a non-standard version of react-scripts'
   )
+  .option(
+    '--layout <layout>',
+    'custom project layout config. e.g. src=fontend',
```
Comment (Leonya) [inline packages/create-react-kotlin-app/createReactApp.js]: Typo here: `fontend`

**Comment by huston007** [comment]
@zeroify Thank you for contributing! Am I getting it right that the motivation is to have multiple "src" folders in the project root?

**Comment by zeroify** [comment]
> @zeroify Thank you for contributing! Am I getting it right that the motivation is to have multiple "src" folders in the project root?

@huston007 yes, so that, i can do the following:
.
|-common
|-server
|-client

react kotlin project and server project share the "common" folder

**Comment by huston007** [comment]
@zeroify Okay, great! I need some time to check how this works. Thank you again.

File: packages/create-react-kotlin-app/createReactApp.js (hunk 1 above)
Comment (zeroify) [reply packages/create-react-kotlin-app/createReactApp.js]: @Leonya not understand your idea, `fontend` is too typical, what your suggestion is. thanks!

Comment (Leonya) [reply packages/create-react-kotlin-app/createReactApp.js]: I mean it should be `frontend`, not `fontend`.

File: packages/react-scripts/template/src/index/index.kt [hunk 2]
```diff
@@ -6,7 +6,7 @@ import react.dom.*
 import kotlin.browser.*
 
 fun main(args: Array<String>) {
-    requireAll(require.context("src", true, js("/\\.css$/")))
+    requireAll(require.context("{{src}}", true, js("/\\.css$/")))
```
Comment (huston007) [inline packages/react-scripts/template/src/index/index.kt]: I'm afraid this kind of syntax doesn't work. Haven't you tried to run your changes locally via "yarn start" in root directory?

`Module not found: Can't resolve '{{src}}' in '~/.../create-react-kotlin-app/packages/react-scripts/node_modules/.cache/kotlin-webpack'`

File: packages/react-scripts/scripts/init.js [hunk 3]
```diff
@@ -19,12 +19,45 @@ const path = require('path');
...
+    return fs
+      .readdirSync(filePath, { withFileTypes: true })
+      .filter(item => ['node_modules', '.git'].indexOf(item.name) == -1)
+      .forEach(item => format(path.join(filePath, item.name), params));
+  }
+  const buffer = fs.readFileSync(filePath);
+  let changed = false;
+  const content = buffer.toString().replace(/\{\{(\w+)\}\}/g, function(m, w) {
+    changed = true;
+    return params.hasOwnProperty(w) ? params[w] : w;
```
Comment (huston007) [inline packages/react-scripts/scripts/init.js]: There is a ESLint issue reported: `Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins`. We couldn't have a successful build until ESLint checks are failed

File: packages/react-scripts/template/src/index/index.kt (hunk 2 above)
Comment (zeroify) [reply packages/react-scripts/template/src/index/index.kt]: I‘ve tried to run `npm start` `npm run build` `yarn start` `yarn build` under window, it is fine.

the step as following:
1. using `npm pack` command, pack the package create-react-kotlin-app and react-scripts
2. using `npm install -g` to install create-react-kotlin-app-2.0.0.tgz 
3. under path `D:\`, run `create-react-kotlin-app test --layout src=frontend` to create project
4. `cd test`, tesing `npm start` `npm run build` `yarn start` `yarn build`

**Pull request closed without merging**